geometry_msgs/Pose pose
uint32 config
float32 theta
int32 action